sauce/helpers/toolkit.js

```diff
@@ -0,0 +1,72 @@
+export function controllerLookup(controllerName) {
+  return containerLookup(`controller:${controllerName}`);
+}
+
+export function componentLookup(componentName) {
+  return containerLookup(`component:${componentName}`);
+}
+
+export function scheduleOnce(onEvent, handler) {
```

Comment:
In my mind, I wanted to use these instead of talking to Ember methods directly because I thought it would be easier to test/mock but it may be more of a pain than it's worth.